docker-compose.yml

```diff
@@ -0,0 +1,5 @@
+version: '2.4'
```

Comment:
The Docker Compose file isn't used by Travis, so I suppose it could be removed from the PR?

Reply:
I think the question is whether this might be potentially useful to anyone else as an example, even if it's not required by Travis.

Reply:
Not really. Removing